### ISSUE TITLE
[Bug]: Chrome Extension with Edge App ID on Blocked Websites

### ISSUE THREAD

**Issue by SamPCS** [CyberDrain/Check#82]
### Required confirmations before submitting

- [x] **I can reproduce this issue on the latest released version** of Check.
- [x] **I have searched existing issues** (both open and closed) to avoid duplicates.
- [x] I am **not** requesting general support; this is an actual bug report.

### Issue Description

We are noticing idle web pages creating false positive blocks with Check. One of these instances was our hosted ScreenConnect instance being run as an Edge Web App. Interestingly, the blocked page for this false positive shows "chrome-extension://knepjpocdagponkonnbggpcnhnaikajg" 

Steps to reproduce:
1. Open Edge web Application of ScreenConnect
2. Idle until Check triggers a block
3. Observe message

<img width="1006" height="483" alt="Image" src="https://github.com/user-attachments/assets/a7c543b5-adf5-49c8-93b2-067ba40b1d52" />

### Extension Version

1.0.5

### Rules Version

1.0.6

### Relevant Logs / Stack Trace

```plaintext

```
